petab/core.py

```diff
@@ -349,21 +372,19 @@ def get_sigmas(sbml_model, remove=False):
     return sigmas
 
 
-def get_noise_distributions(measurement_df):
+def get_noise_distributions(measurement_df: pd.DataFrame) -> dict:
     """
-    Returns dictiontary of cost definitions per observable, if specified.
+    Returns dictionary of cost definitions per observable, if specified.
+
     Looks through all parameters satisfying `sbml_parameter_is_cost` and
-    return dictionary of key: observableId, value: cost definition.
+    return as dictionary.
 
-    Parameters
-    ----------
+    Parameters:
```

Comment:
is that in line with python docstring formatting? don't know.

Reply:
In AMICI we are using the colon-version. Both are valid. I prefer the latter since it means less typing.